src/patternfly/components/Dropdown/examples/dropdown-groups-example.hbs

```diff
@@ -0,0 +1,5 @@
+{{#> dropdown id="dropdown-basic-example-expanded" dropdown--IsExpanded="true" dropdown--HasToggleIcon="true" dropdown--IsGroups="true"}}
```

Comment:
maybe change the ID to dropdown-group-example-expanded.
and ...
dropdown--IsGroups , to dropdown--HasGroups.

Reply:
good catch. to stay consistent with the other menus, changed it to `dropdown--IsGroupsMenu`